django_project/lesson/templates/worksheet/detail.html

```diff
@@ -341,6 +341,15 @@ <h3 class="custom-heading" style="margin-left:-40px"><strong>Further reading: </
             </ul>
         </div>
     </div>
+    <div class="row">
+        <div class="col-lg-12 text-center" style="margin-bottom: 10px;">
+            {% if worksheet.external_data %}
+            <p>Click <a href="{{ worksheet.external_data.url }}">here</a> to
```

Comment:
Make the link opening in a new tab, to avoid losing your current tab

Reply:
When you click it only downloads the file and the current tab remains unchanged

Reply:
Hum, maybe you are right. ok so ;-)